packages/honeycomb-opentelemetry-web/examples/hello-world-custom-exporter/index.js

```diff
@@ -0,0 +1,77 @@
+import {
+  getDefaultTraceExporter,
+  HoneycombWebSDK,
+} from '@honeycombio/opentelemetry-web';
+import { trace } from '@opentelemetry/api';
+import { getWebAutoInstrumentations } from '@opentelemetry/auto-instrumentations-web';
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-base';
+
+const configDefaults = {
+  ignoreNetworkEvents: true,
+};
+
+const main = () => {
+  const hcoConfig = {
+    // defaults to sending to US instance of Honeycomb
+    // endpoint: "https://api.eu1.honeycomb.io/v1/traces", // uncomment to send to EU instance
+    apiKey: 'api-key-goes-here',
```

Comment:
only the endpoint, apiKey and dataset (classic environments) fields are used to create the default exporter.

Reply:
It might be a clearer example to move this inline with other config option?